Storage Provider Incubation Center/README.md

```diff
@@ -47,11 +47,11 @@
 
  ### Is it suitable for you? 
 
- <img width="1288" alt="Rules and Requirements" src="https://user-images.githubusercontent.com/75723122/125607988-7553fec8-83c4-4bc8-bcce-8f19c57aefc0.png">
+Rules and Requirements.png![Rules and Requirements](https://user-images.githubusercontent.com/75723122/127090179-75fee051-b36f-475d-9029-4c996889b2f0.png)
 
  #### Storage Provider Incubation Center Program (Track 1)
 
- The storage provider incubation center do welcome new or small storage providers to join immediately, you need to seal at least 64 sectors per day (2TiB）to join this program, when a storage provider's storage power reached 1PiB or has participated this program over 90days, then the storage providers has to withdraw from this program, and the others who meet the requirements can be joined.
+ The storage provider incubation center do welcome new or small storage providers to join immediately, you need to seal at least 64 sectors per day (2TiB）to join this program, when a storage provider's storage power reached 1PiB or run to the end of the certain phase, then the storage providers has to withdraw from this program, and the others who meet the requirements can be joined.
 
  The first term of Storage Provider Incubation Center started from **2021/07/26 - 2021/10/24**. If you want to join this program, please register it here [Application Form](http://venusteam.mikecrm.com/1lmpQtj)，please fill as much content as you can so we can review and give feedback to you within a short time. Please be aware, due to the software and hardware varied a lot, we have to learn more about your application before making the right choice. 
 
```

Comment:
The first cohort of the Storage Provider Incubation Center is targeted for **2021/07/26 - 2021/10/24**. If you want to join this program, please register here [Application Form](http://venusteam.mikecrm.com/1lmpQtj) and  fill out as much detail as possible so we can accelerate our response to your submission. Please be aware, due to the varies possible software and hardware combinations, we will need to review your configuration to verify the feasibility to participate. We will work with you and guide you to configure the right permutation.